docs/builtin_workflows.rst

```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
+
+.. code-block::
+
+   qsirecon/
+      derivatives/
+         qsirecon-<suffix>/
+            sub-<label>/
+               dwi/
+                  sub-<label>_space-T1w_seg-<label>_dseg.nii.gz
+                  sub-<label>_space-T1w_seg-<label>_dseg.mif.gz
+                  sub-<label>_space-T1w_seg-<label>_dseg.json
+                  sub-<label>_space-T1w_seg-<label>_dseg.tsv
+
+
+.. _custom_atlases:
+
+Using custom atlases
+^^^^^^^^^^^^^^^^^^^^
+
+It's possible to use your own atlases provided you organize the atlases into BIDS-Atlas datasets.
+Users can control which atlases are used with the ``--atlases`` and ``--datasets`` parameters.
+
+The nifti images should be registered to the
+`MNI152NLin2009cAsym <https://github.com/PennLINC/qsirecon/blob/main/qsirecon/data/mni_1mm_t1w_lps.nii.gz>`_
+included in ``qsirecon``.
+It is essential that your images are in the LPS+ orientation and have the sform zeroed-out in the header.
+**Be sure to check for alignment and orientation** in your outputs.
+
+
+*********************
+Connectivity Measures
+*********************
+
+Connectivity measures are bundled together in binary ``.mat`` files,
+rather than as atlas- and measure-specific tabular files.
+
+.. warning::
+
+   We ultimately plan to organize the connectivity matrices accoring to the BIDS-Connectivity BEP,
+   wherein each measure from each atlas is stored in a separate file.
+
+   Therefore, this organization will change in the future.
+
+.. code-block::
+
+   qsirecon/
+      derivatives/
+         qsirecon-<suffix>/
+            sub-<label>/[ses-<label>/]
+               dwi/
+                  <source_entities>_connectivity.mat
+
+The ``.mat`` file contains a dictionary with all of the connectivity measures specified
+by the recon spec for all of the different atlases specified by the user.
+
+For example, in the case where a user has selected a single atlas (``<atlas>``) and
+the recon spec specifies a single connectivity measure (``<measure>``),
+the ``.mat`` file will contain the following keys:
+
+.. code-block::
+
+   command                                # The command that was run
+   atlas_<atlas>_region_ids               # The region ids for the atlas (1 x n_parcels array)
+   atlas_<atlas>_region_labels            # The region labels for the atlas (1 x n_parcels array)
+   atlas_<atlas>_<measure>_connectivity   # The connectivity matrix for the atlas and measure (n_parcels x n_parcels array)
+
+
+MRtrix3 Connectivity Measures
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+MRtrix3 connectivity workflows produce 4 structural connectome outputs for each atlas.
+The 4 connectivity matrix outputs are
+
+   * *radiusN.count.connectivity*: raw streamline count based matrix
+   * *sift.radiusN.count.connectivity*: sift-weighted streamline count based matrix
+   * *radiusN.meanlength.connectivity*: a matrix containing mean length of raw streamlines
+   * *sift.radiusN.meanlength.connectivity*: a matrix containing mean length of sifted output
```

Comment:
```suggestion
   * *atlas_<atlas>_radius<N>.count.connectivity*: raw streamline count based matrix
   * *atlas_<atlas>_sift.radius<N>.count.connectivity*: sift-weighted streamline count based matrix
   * *atlas_<atlas>_radius<N>.meanlength.connectivity*: a matrix containing mean length of raw streamlines
   * *atlas_<atlas>_sift.radius<N>.meanlength.connectivity*: a matrix containing mean length of sifted output
```

I think that's what this should be, but I could be wrong.

Reply:
I think you're right